groups/ecs/ecs.tf

```diff
@@ -0,0 +1,47 @@
+resource "aws_ecs_cluster" "grafana_cluster" {
+  name = "${var.service}-cluster"
+}
+
+resource "aws_ecs_task_definition" "grafana_task" {
+  family                   = "${var.service}-task"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "256"
```

Comment:
I would pull these values out as variables. We can default those variables to these values but we should have that flexibility